### PR TITLE
Add --task-log option in local mode

### PR DESCRIPTION
**Pull request by yoyama**
This patch supports --task-log option in local mode. (Issue #783)
I added LogModule to DigdagEmbed in class Run.

### CODE REVIEW

File: digdag-docs/src/command_reference.rst [hunk 1]
```diff
@@ -106,6 +106,11 @@ Options:
 
   Example: ``--max-task-threads 5``
 
+:command:`-O, --task-log DIR`
+  Store task logs to this directory.
+
+  Example: ``--task-log digdag.log``
```
Comment (muga) [inline digdag-docs/src/command_reference.rst]: `digdag.log` looks like file name. we'd better to change it with directory-like name. for example, `log/tasks` or something. what do you think?

Comment (yoyama) [reply digdag-docs/src/command_reference.rst]: @muga Thank you for your pointing out.
I agree with you and fix it.
If there are something other issues, please let me know.

Best regards,
  You Yamagata

**Comment by yoyama** [comment]
Sorry for my late response.
I have already fixed the document.
Could you confirm it?

Best regards,
  You Yamagata

**Comment by muga** [comment]
@yoyama Thank you for updating! Merging..

**Pull request merged**